docker-compose.yml

```diff
@@ -81,3 +81,13 @@ services:
   airflow_worker:
     <<: *airflow
     command: worker
+
+  content-store:
+    image: jats-ingester_airflow_webserver
```

Comment:
this `image` shouldn't be needed `(`build` is used instead) so can be removed?

Reply:
image is used and if the image is not found build is used as a fallback: https://docs.docker.com/compose/compose-file/#image